core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -489,8 +489,7 @@ class QualificationAppInfo(
         val filteredChildren = e.children.map { c =>
           c.filterNot(_.shouldRemove)
         }
-        new ExecInfo(e.sqlID, e.exec, e.expr, e.speedupFactor, e.duration,
-          e.nodeId, e.isSupported, filteredChildren, e.stages, e.shouldRemove, e.unsupportedExprs)
+        e.copy(children = filteredChildren)
```

Comment:
There was a bug in earlier code where udf value was not copied so udf if set would be missed for some Execs. This would fix that.

Reply:
Nice that you also noticed it.
I gazed to that line for a moment and said "oh boy"!